parquet-column/src/test/java/org/apache/parquet/schema/TestTypeBuildersWithLogicalTypes.java

```diff
@@ -205,10 +206,20 @@ public void testBinaryAnnotations() {
     }
   }
 
+  @Test
+  public void testFloat16Annotations() {
+    LogicalTypeAnnotation[] types = new LogicalTypeAnnotation[] {float16Type()};
+    for (final LogicalTypeAnnotation logicalType : types) {
```

Comment:
It seems that we don't need this for loop?

Reply:
delete necessary loop, thanks.